assets/scss/_home.scss

```diff
@@ -576,8 +576,38 @@
         }
     }
 
-    #complextopics .container {
-        margin-top: auto;
+    #complextopics {
+        @media (min-width: 1600px) {
+            max-height: 643px;
+        }
+    
+        .container {
+            margin-top: auto;
+        }
+
+        .section-img {
+            img {
+                margin-top: -6em;
```

Comment:
The negative margin here pushes the character too high on extremely small sizes and doesn't seem needed on any other sizes. Please remove.